department-of-veterans-affairs/caseflow
# Fix Case Details Page Second Actions Dropdown Displaying

Resolves [APPEALS-24904](https://jira.devops.va.gov/browse/APPEALS-24904), [APPEALS-27506](https://jira.devops.va.gov/browse/APPEALS-27506), [APPEALS-27966](https://jira.devops.va.gov/browse/APPEALS-27966)
[APPEALS-26134](https://jira.devops.va.gov/browse/APPEALS-26134) and [APPEALS-28243](https://jira.devops.va.gov/browse/APPEALS-28243)

# Description
Value Statement: As a Caseflow QA, BA, and Developer, I need to ensure that scenario 5 - As a Special Case Movement User, assigning a case in a VLJ's possession to an attorney is working as intended outlined in the requirements so that it is functional in all environments.

## Acceptance Criteria


## Testing
- Login to: Moon Walkers Demo Environment

- Change user to [BVARDUNKLE (VACO)](https://mw.caseflowdemo.com/queue/appeals/082250264#BVARDUNKLE%20(VACO))
- Select task action dropdown: Case Movement: Cancel blocking tasks and advance to judge
- Fill out Reassign page
- Confirm cancellation and reassignment on modal
- Green banner displays on Case Details Page
- On Case Details Page, we NO see second actions dropdown

![scenario1](https://github.com/department-of-veterans-affairs/caseflow/assets/110848569/a9be58b4-ab1a-4356-8e72-32a21c45f09f)
